src/main/default/lwc/mainTitle/__tests__/mainTitle.test.js

```diff
@@ -0,0 +1,86 @@
+import { createElement } from 'lwc';
+import MainTitle from 'c/mainTitle';
+
+function createTestComponent() {
+  const element = createElement('c-main-title', {
+    is: MainTitle
+  });
+  document.body.appendChild(element);
+
+  return element;
+}
+
+// Helper function to wait until the microtask queue is empty.
+function allPromisesResolution() {
+  // eslint-disable-next-line no-undef
+  return new Promise((resolve) => setImmediate(resolve));
+}
+
+jest.mock(
+  '@salesforce/label/c.cookbook_MainTitle',
+  () => ({ default: 'Hi, what do you need help with?' }),
+  {
+    virtual: true
+  }
+);
+
+describe('c-main-title', () => {
+  afterEach(() => {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+  });
+
+  it('should display the correct value of the title', async () => {
+    const element = createTestComponent();
+    const expectedTitle = 'Expected Title';
+    element.title = expectedTitle;
+
+    await allPromisesResolution();
+    const titleNode = element.shadowRoot.querySelector(
+      'h1.slds-text-heading_large'
+    );
+    expect(titleNode).not.toBeNull();
+    expect(titleNode.textContent).toBe(expectedTitle);
+    expect(element).toBeAccessible();
```

Comment:
According to `sa11y-jest` documentation, you must `await` when invoking `toBeAccessible`.